pygama/dsp/_processors/get_multi_local_extrema.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+from numba import guvectorize, njit
+from pygama.dsp.errors import DSPFatal
+
+
+
+@guvectorize(["void(float32[:], float32, float32[:], float32[:], float32[:], float32[:], float32[:], int32[:])",
+              "void(float64[:], float64, float64[:], float64[:], float64[:],float64[:], float64[:], int32[:])",
```

Comment:
Our convention is to use only floats, even if the parameters are integers. The reason is that we use NaN return values to track failures and integers cannot be NaN. I suggest changing the int32 into a float32 in the first line and into a float64 in the second line